src/common/tests/CommonUtilsUT.cpp

```diff
@@ -236,15 +242,15 @@ TEST_F(CommonUtilsTest, ExecuteCommandWithStdErrOutput)
     char* textResult = nullptr;
 
     EXPECT_EQ(127, ExecuteCommand(nullptr, "hh", false, true, 100, 0, &textResult, nullptr, nullptr));
-    EXPECT_NE(nullptr, strstr(textResult, "sh: 1: hh: not found\n"));
+    EXPECT_NE(nullptr, strstr(textResult, "not found")) << "Text result: '" << textResult << "'";
```

Comment:
>EXPECT_NE(nullptr, strstr(textResult, "not found")) << "Text result: '" << textResult << "'"; [](http://example.com/codeflow?start=4&length=93)

If (Mariner)
{
EXPECT_NE(nullptr, strstr(textResult, "......"));
}
else
{
EXPECT_NE(nullptr, strstr(textResult, "sh: 1: hh: not found\n"));
}

Reply:
No <<, please. Leave the original line as-is, make the new line like it.

Reply:
In fact, keep the one line and inside, instead of:

"sh: 1: hh: not found\n"

Do:

IsMarinerDistro() ? "new one" : "sh: 1: hh: not found\n"